heatmap/thread_safety.md

```diff
@@ -0,0 +1,71 @@
+A `Heatmap` is made up of a collection of `Histogram`s covering the `span` of
```

Comment:
I think this is great, but not immediately visible on docs.rs - would probably be better to put this as doc comments at the top of lib.rs

Reply:
Oh good point, let me do that.